### PR TITLE
更正pip安装mysql-connector-python步骤

### PR DESCRIPTION
**Pull request by DecKen**
pip安装mysql-connector-python需要 --allow-external参数
测试环境:
1. python2.7.9 ubuntu14.04 pip 6.0.6
2. python2.7.5 fedora20 pip1.5.6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Web
 
 ```
 apt-get install python-dev
-pip install tornado u-msgpack-python jinja2 chardet requests mysql-connector-python redis pbkdf2 pycrypto
+pip install tornado u-msgpack-python jinja2 chardet requests redis pbkdf2 pycrypto
+pip install mysql-connector-python --allow-external mysql-connector-python
```
Comment (binux) [inline README.md]: mysql-connector-python 不用写两遍

Comment (DecKen) [inline README.md]: 安装日志, 测试环境:
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.1 LTS"

pip 1.5.4 from /usr/lib/python2.7/dist-packages (python 2.7)
```
$sudo pip install mysql-connector-python                 
Downloading/unpacking mysql-connector-python
  http://pypi.dglinux.com/simple/mysql-connector-python/ uses an insecure transport scheme (http). Consider using https if pypi.dglinux.com has it available
  Could not find any downloads that satisfy the requirement mysql-connector-python
  Some externally hosted files were ignored (use --allow-external mysql-connector-python to allow).
Cleaning up...
No distributions at all found for mysql-connector-python
Storing debug log for failure in /root/.pip/pip.log
```

```
$sudo pip install mysql-connector-python --allow-external

Usage:   
  pip install [options] <requirement specifier> ...
  pip install [options] -r <requirements file> ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

--allow-external option requires an argument
```

```
$sudo pip install mysql-connector-python --allow-external mysql-connector-python
Downloading/unpacking mysql-connector-python
  http://pypi.dglinux.com/simple/mysql-connector-python/ uses an insecure transport scheme (http). Consider using https if pypi.dglinux.com has it available
  mysql-connector-python an externally hosted file and may be unreliable
  Downloading mysql-connector-python-2.0.3.zip (275kB): 275kB downloaded
  Running setup.py (path:/tmp/pip_build_root/mysql-connector-python/setup.py) egg_info for package mysql-connector-python
    
Installing collected packages: mysql-connector-python
  Running setup.py install for mysql-connector-python
    
Successfully installed mysql-connector-python
Cleaning up...
```

**Comment by aprikyblue** [comment]
已测试，的确要这样写

如果只写一遍，会报错
> \# pip install mysql-connector-python
Downloading/unpacking mysql-connector-python
  Could not find any downloads that satisfy the requirement mysql-connector-python
  Some externally hosted files were ignored (use --allow-external mysql-connecto-python to allow).

> \# pip install --allow-external mysql-connector-python
You must give at least one requirement to install (see "pip help install")

> \# pip install mysql-connector-python --allow-external mysql-connector-python
Downloading/unpacking mysql-connector-python
  Downloading mysql-connector-python-2.0.4.zip (277kB): 277kB downloaded
  Running setup.py (path:/tmp/pip_build_root/mysql-connector-python/setup.py) egg_info for package mysql-connector-python

Installing collected packages: mysql-connector-python
 Running setup.py install for mysql-connector-python

Successfully installed mysql-connector-python

第一个是用来指定要安装的包名列表，第二个mysql-connector-python指定是给--allow-external的

**Comment by binux** [comment]
有个参数叫 --allow-all-external
并不是所有版本的 pip 都需要这个参数
当它提示你的时候, 那就加上好了.

**Comment by ZMOM1031** [comment]
真正正确的应该是先下载安装包，本地离线安装
https://pypi.python.org/pypi/mysql-connector-python
wget http://url
pip install mysql-connector-python-2.0.4.zip

时隔半年，再来配置这个服务终于成功了，之前一直用cron每天执行命令，果然还是PHP好啊

**Pull request closed without merging**